src/js/InfoBox.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import './InfoBox.css';
+
+class InfoBox extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentPrice: null,
+      changeDollar: null,
```

Comment:
Slightly minor point about naming is that these now sound like methods. Maybe "differenceDollars" and "differencePercentage" or something?